### PR TITLE
Feature css polish

### PR DESCRIPTION
**Pull request by Brenisec**
Added Spanish deck button. cleaned ui to fit said button. button fade for better understanding of where buttons are.

### CODE REVIEW

File: src/main/java/views/AcesUp/AcesUp.flt.html [hunk 1]
```diff
@@ -110,7 +116,11 @@
 </table>
 
 <div class="removers">
-<BR><button id="Remove1">Remove Top</button><button id="Remove2">Remove Top</button><button id="Remove3">Remove Top</button><button id="Remove4">Remove Top</button>
```
Comment (akinde) [inline src/main/java/views/AcesUp/AcesUp.flt.html]: Better change and it looks alot better too. Conserves space.

File: src/main/java/assets/css/AcesUp.css [hunk 2]
```diff
@@ -16,64 +18,165 @@ text-align:center;
...
-}
+/*button:hover{
+  background-color: white;
+}*/
 
-.fromTo{
-	width: 50px;
+.menuButton{
+  background-color: #319df6;
+  transition: top .5s, background-color .5s;
```
Comment (akinde) [inline src/main/java/assets/css/AcesUp.css]: Shorter and more effective, nice.

File: src/main/java/assets/css/AcesUp.css [hunk 3]
```diff
@@ -16,64 +18,165 @@ text-align:center;
...
+  padding-left: 23px;
+  padding-right: 23px;
+  border: 3px;
+  border-radius: 10px;
+  border-style: solid;
+  border-color: black;
+}
 
+.removers{
+	text-align: center;
```
Comment (akinde) [inline src/main/java/assets/css/AcesUp.css]: Has a tab but isnt critical.

File: src/main/java/views/AcesUp/AcesUp.flt.html [hunk 4]
```diff
@@ -4,37 +4,43 @@
...
         <td >Column:3</td>
-=======
-<body
-<table>
+<!-- ======= -->
+<!-- <table>
     <tr class="title">
         <td class="hide">Column0</td>
         <td class="hide">Column1</td>
         <td class="hide">Column2</td>
```
Comment (akinde) [inline src/main/java/views/AcesUp/AcesUp.flt.html]: You probably can remove this because really it's not being used.

**Comment by akinde** [comment]
I really like your style, lets see what we can do with this. Yes *snaps finger*

File: src/main/java/assets/css/AcesUp.css [hunk 5]
```diff
@@ -2,7 +2,7 @@ body {
   font-family:  Helvetica, sans-serif;
   margin: 30px;
   overflow: hidden;
-  background-color: grey;
+  background-color: #f3f5f6;
```
Comment (hutchico) [inline src/main/java/assets/css/AcesUp.css]: So we went from "grey" to "slightly less grey". Nice.

File: src/main/java/assets/css/AcesUp.css [hunk 6]
```diff
@@ -44,11 +40,11 @@ font-weight: bold;
...
-background-color:#306def;
-border-radius: 4px;
-border-color:#010001;
-height: 100px;
-width: 650px;
+  background-color:#306def;
+  border-radius: 4px;
+  border-color:#010001;
+  height: 115px;
+  width: 650px;
```
Comment (gilesni) [inline src/main/java/assets/css/AcesUp.css]: I really enjoy the color changes that you made and feel they add a nice aesthetic to the menu bar

**Pull request merged**